src/lib/create-config.ts

```diff
@@ -661,6 +661,10 @@ export function createConfig(options: IUnleashOptions): IUnleashConfig {
             process.env.UNLEASH_ENVIRONMENTS_LIMIT,
             50,
         ),
+        apiTokens: parseEnvVarNumber(
```

Comment:
I'd wrap it with `Math.max(1, parse...)` if you want to enforce min 1. But check my next comments about the minimum value to think if we need one.

Reply:
Roger.